int32   id
float32 x
float32 y
float32 orientation
float32 v_x
float32 v_y
int32 last_seen
int32 continously_seen
int32 max_continously_seen
bool visualized
